src/pages/careers/_employee-testimonial-carousel.js

```diff
@@ -111,28 +104,24 @@ EmployeeSlide.propTypes = {
 
 const ahmad = {
     name: 'Ahmad Aizuddin, Disaster Recovery Analyst',
-    img_alt: 'Ahmad - Recovery Analyst',
```

Comment:
@nikolaiberestevich-deriv We will need these as this one is for the image itself and not an icon.